features/support/topical_events_helper.rb

```diff
@@ -30,6 +30,10 @@ def sample_document_types_and_titles
     }
   end
 
+  def rummager_response
+    File.read(Rails.root.join('features/fixtures/rummager_response.json'))
```

Comment:
This is an interesting approach, I get that it's tidier and means you don't have 36 lines more in this file but to my mind so thats good, unless there is some super specific reason it has to be a separate file, I'd err away from it and move the resultant somewhat large method to the bottom  (feel free to disagree though)

Reply:
I'm using this fixture in a couple of different places so I pulled it out into a JSON. I can put it into a new rummager test helper class, which might be a bit neater?

Reply:
Never tried this before but there is this: https://api.rubyonrails.org/v5.1/classes/ActiveSupport/Testing/FileFixtures.html

Reply:
Do you know if it would work if the tests are in different directories? This is used in `/features` and `/test/unit`

Reply:
you could set it in a helper as:

```ActiveSupport::TestCase.file_fixture_path='features/fixtures'```

Anyway, not sure how much better it is - just came across this feature